docs/docs/guides/labs/freshness/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Freshness Policies'
+sidebar_position: 100
+unlisted: True
+---
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+## Overview
+
+Freshness policies help you understand which of your assets have materialized recently and which ones are running behind - a key component of asset health. Freshness policies also communicate expectations for data freshness, allowing downstream asset consumers to determine how often assets are expected to be updated.
+
+For example, freshness policies can help identify stale assets caused by:
+
+- Misconfigured `AutomationCondition`s
+- Runs not being scheduled due to an upstream failure
+- Runs taking longer than expected to complete
+
+### Wait, isn't there already a `FreshnessPolicy`?
+Yes, there is an existing `FreshnessPolicy` API that has been deprecated for quite some time. We're opting to reuse the name for the new freshness APIs, which requires us to migrate off the legacy API.
+
+To avoid naming conflicts with the legacy API during this migration, we've prefixed the new APIs, class names and method args with "internal" (for example, `InternalFreshnessPolicy` and the `internal_freshness_policy` arg on the `@asset` decorator). Once the migration is complete and the new APIs exit preview mode, the "internal" prefix will be removed and `FreshnessPolicy` will be the name of the new APIs.
+
+## Types of Freshness Policies
+
+Currently, we support time window-based freshness policies, which are suitable for most use cases. We plan to add more policy types in the future.
+
+### Time Window Policy
+
+A time window freshness policy is useful when you expect an asset to have new data or be recalculated with some frequency.
```

Comment:
```suggestion
A time window freshness policy is useful when you expect an asset to have new data, or be recalculated with some frequency. An asset that does not meet this condition will be considered failing its freshness policy. You can set an optional warning window on a freshness policy; if the asset does not successfully materialize within this window, it will enter a `warning` freshness state.

For example, the policy below states that there should be a successful materialization of the asset at least every 24 hours for it to be considered fresh, with a warning window of 12 hours:
```